anaUltraScurve.py

```diff
@@ -210,16 +226,61 @@ def overlay_fit(VFAT, CHAN):
     trimrange_list[event.vfatN][event.vfatCH] = event.trimRange
     pass
 
+# Determine hot channels
+from anautilities import isOutlierMADOneSided
+import numpy as np
 if options.SaveFile:
+    print 'Determining hot channels'
+    masksDead = []
+    masksHot = []
     masks = []
+    for vfat in range(0, 24):
+        trimValue = np.zeros(128)
+        dead = np.zeros(128)
+        for ch in range(0, 128):
+            # Get fit results
+            threshold[0] = scanFits[0][vfat][ch]
+            noise[0] = scanFits[1][vfat][ch]
+            pedestal[0] = scanFits[2][vfat][ch]
+            # Identify dead channels
+            dead[ch] = (threshold[0] == 8 and pedestal[0] == 8)
+            # Compute the value to apply MAD on for each channel
+            trimValue[ch] = threshold[0] - options.ztrim * noise[0]
+        masksDead.append(dead)
+        # Determine outliers
+        hot = isOutlierMADOneSided(trimValue, thresh=options.zscore,
+                                   rejectHighTail=False).flatten()
```

Comment:
Okay, I see, don't do this.  We should make the package as uniform as possible, having two different ways of implementing this usage will be harder to maintain over the longterm.

Please either:
- Change your implementation to match what is done in anaUltraThreshold.py, or
- Change anaUltraThreshold.py to use the implementation shown here.

I strongly favor the first option.  To me the use of "flatten" is unnecessary and confusing.  Please update.